xml/System/Range.xml

```diff
@@ -54,22 +54,14 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>Represents a range that has start and end indexes.</summary>
+    <summary>Represent a range has start and end indexes.</summary>
     <remarks>
-      <format type="text/markdown"><![CDATA[
-
-## Remarks
-
-`Range` is used by the C# compiler to support the range syntax:
-
-```csharp
-int[] someArray = new int[5] { 1, 2, 3, 4, 5 };
-int[] subArray1 = someArray[0..2];               // { 1, 2 }
-int[] subArray2 = someArray[1..^0];              // { 2, 3, 4, 5 }
-```
-
-          ]]></format>
-    </remarks>
+            Range is used by the C# compiler to support the range syntax.
```

Comment:
Figure out where this is coming from.

Reply:
Coming from .NET Standard and .NET Framework frameworks of https://www.nuget.org/packages/Microsoft.Extensions.VectorData.Abstractions. See [source code](https://github.com/microsoft/semantic-kernel/blob/4724dbc8f330fc773cc6aba44c39fa2027aa1bf3/dotnet/src/InternalUtilities/src/System/IndexRange.cs#L165).

Reply:
@huangmin-ms This is similar to the [problem we encountered for ObsoleteAttribute](https://github.com/dotnet/dotnet-api-docs/pull/10395#discussion_r1758128787) docs. I forgot how we resolved that one. I see the [source code](https://github.com/dotnet/extensions/blob/fa2d656fba04cdd7ac06cea6dadbc9aee73b1d5d/src/LegacySupport/ObsoleteAttribute/ObsoleteAttribute.cs#L10-L17) hasn't changed, so did you special case it in the mdoc code?